packages/runtime/container-runtime/src/summary/summarizerTypes.ts

```diff
@@ -442,7 +442,7 @@ export interface ISummarizeHeuristicRunner {
 
 type ISummarizeTelemetryRequiredProperties =
 	/** Reason code for attempting to summarize */
-	"reason";
+	"summarizeReason";
```

Comment:
What about `IOnDemandSummarizeOptions` as well?

Reply:
I thought about that but decided to leave that to reason because that is a public facing API. So, `reason` already conveys its for summarize.